### PR TITLE
Point to quyykk's editor

### PR DESCRIPTION
**Pull request by TheGiraffe3**
## Summary

Changed the link in MapData to point to quyykk's editor, as the normal one is years out of date.

### CODE REVIEW

File: wiki/MapData.md [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 # Introduction
 
-A useful tool for creating planets and systems is the [Endless Sky editor](https://github.com/endless-sky/endless-sky-editor). This most especially helps with the positioning of systems and the distance/period of planets in a system.
+An useful tool for creating planets and systems is the [Endless Sky editor](https://github.com/quyykk/plugin-editor). This most especially helps with the positioning of systems and the distance/period of planets in a system.
```
Comment (tibetiroka) [inline wiki/MapData.md:16]:
```suggestion
A useful tool for creating planets and systems is the [Endless Sky editor](https://github.com/quyykk/plugin-editor). This most especially helps with the positioning of systems and the distance/period of planets in a system.
```

Comment (tibetiroka) [reply wiki/MapData.md:16]: https://www.merriam-webster.com/grammar/is-it-a-or-an

**Comment by TomGoodIdea** [comment]
If we're not planning to update the "official" editor, its repo should be marked as archived IMO, or at least there should be a visible note that it's no longer maintained.

**Comment by mOctave** [comment]
Since Quyykk seems to have abandoned the plugin editor and mostly moved away from ES, should we be reconsidering this? Or is there a way to transfer ownership of the editor to the endless-sky org?

**Comment by mOctave** [comment]
By the way, Quyykk's editor is now old enough and outdated enough that it no longer runs on MacOS. That means that, as of right now, it is no longer better than the official one for anyone running MacOS (still not worse, because the old one needs custom building too, but...)

**Comment by TheGiraffe3** [comment]
> By the way, Quyykk's editor is now old enough and outdated enough that it no longer runs on MacOS. That means that, as of right now, it is no longer better than the official one for anyone running MacOS (still not worse, because the old one needs custom building too, but...)

It works on my Mac.

**Comment by TheGiraffe3** [comment]
It doesn't look like this will be merged, at least not until if/when quyykk decides to transfer the editor. Closing for now.

**Pull request closed without merging**